Package.swift

```diff
@@ -1,11 +1,11 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Header comments should be consistent with project patterns.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)